corehq/motech/repeaters/tests/test_models.py

```diff
@@ -113,21 +114,16 @@ def setUp(self):
         self.repeater = FormRepeater(
             domain=DOMAIN,
             url=url,
-            connections_settings_id=conn.id
         )
-        self.repeater.save()
-        self.sql_repeater = SQLRepeater.objects.create(
+        self.repeater.save(sync_to_sql=False)
+        self.sql_repeater = SQLFormRepeater(
             domain=DOMAIN,
             repeater_id=self.repeater.get_id,
             connection_settings=conn,
         )
+        self.sql_repeater.save(sync_to_couch=False)
```

Comment:
This is required because some of the tests check certain things which are populated by the sync logic. Like for `test_create_connection_settings`, it requires `connection_settings_id` to be None in the beginning and only be populated when we call `connection_settings` attribute. But when we are syncing to SQL we are accessing this property which is resulting in this property getting saved to the couch as well. 

After we have completely moved to SQL this test would not make sense, but I am not sure if we should be removing the test here.